dot/core/messages.go

```diff
@@ -24,19 +24,20 @@ import (
 
 // HandleTransactionMessage validates each transaction in the message and
 // adds valid transactions to the transaction queue of the BABE session
```

Comment:
Would be nice to have a description that the bool return means there is extrinsic to propagate.

Reply:
good point... Updated.